Resources/Server Info/Guidebook/Med/Chemistry.xml

```diff
@@ -0,0 +1,38 @@
+<Box>
+<GuideEntityEmbed Entity="chem_dispenser"/>
+</Box>
+The reagent grinder is a multipurpose device used in multiple areas of work, be it hydroponics, cooking, or in this case, chemistry. Within the context of chemistry, you will most commonly use this device to grind [color=#7e009e]plasma[/color], or to grind up mystery pills to see what's inside. To use the machine, first simply insert a sufficently sized container, and then the object(s) you would like to grind or juice.
+<Box>
+<GuideEntityEmbed Entity="KitchenReagentGrinder"/>
+</Box>
+The chem master 4000 is not a required machine if you're only looking to drink or inject medicines, however if you find yourself in a position where you would need to make pills/bottles of medicine (or perhaps a poison like [color=#a1000b]bleach[/color]), this is the machine for the job. To use the machine, simply insert a sufficently sized container full of the reagent you'd like to turn into a pill/bottle. Next, fill the buffer of the machine with the aforementioned reagent from the container, and repeat until you have a satisfactory amount of reagents inside the buffer. Don't worry about accidental mixing of reagents inside the buffer, as they do not mix when inside the buffer. The buffer can also hold an unlimited amount of reagents, so fill it with a vast library of chemicals if you so desire. Once you have enough reagents in the buffer, insert a pill canister or bottle into the machine, adjust the dosage and amount of pills/units of reagent to your desire, and click the "dispense" button. A bottle/pill container with the requested items will appear in your active hand. That's how you use the machine.
+<Box>
+<GuideEntityEmbed Entity="chem_master"/>
+</Box>
+## Basic medicinal chemical mixes to know
+- [color=#731024]Inaprovaline[/color], an emergency airloss cure, is made with 1u [color=#808080]oxygen[/color], 1u [color=#3e464a]carbon[/color], and 1u [color=#ffffff]sugar[/color]
+- [color=#ffaa00]Bicardine[/color], which heals 4 brute damage per unit, is made with 1u [color=#731024]inaprovaline[/color], and 1u [color=#3e464a]carbon[/color].
+- [color=#bf3d19]Kelotane[/color], a light and un-overdose-able burn (heals 2u of burn/unit) medication, is made with 1u [color=#364266]silicon[/color], and 1u [color=#3e464a]carbon[/color].
+- [color=#0041a8]Dexalin[/color], the precursor to [color=#4da0bd]Dexalin+[/color], is made with 5u [color=#7e009e]plasma[/color], and 2u [color=#808080]oxygen[/color]. The plasma is not consumed in the reaction.
+- [color=#4da0bd]Dexalin+[/color], a hugely helpful airloss and bloodloss cure (heals 7u asphyxiation and 6u bloodloss/unit), is made with 1u [color=#434b4d]iron[/color], 1u [color=#0041a8]Dexalin[/color], and 1u [color=#3e464a]carbon[/color].
```

Comment:
Dexalin and dexalin plus heal bloodloss damage but not loss of blood. If this distinction can be made somewhere (and kept as a standard when writing medical material), that would help reduce confusion.